docs/_docs/Unofficial Sidecar Academy/Sidecar-Academy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: LDSA Unofficial Sidecar Academy
+category: Unofficial Sidecar Academy
+order: 1
+---
+
+Contents:
+- [Course Overview](#course-overview)
+- [What will be covered](#what-will-be-covered)
+- [What is the same and what is different](#what-is-the-same-and-what-isdifferent)
+
+---
+
+# Course overview
+Over the last few years, many people have wanted to but not been able to get into the LDSSA. As of now, we have not been able to offer them any community-based alternatives which leaves them with the usual options of lonely online courses. While the course material, portal, and access to instructors are definitely useful things, the vast majority of the value provided to the students is in the community aspect. The feeling that you're not going it alone is incredibly powerful.
+
+Since 2020, we have started to offer the LDSA Unofficial Sidecar Academy, a community-based lightweight version of the LDSA and is meant to provide all of those who didn't have a chance to get into the 2020 LDSSA, a concrete next step in their pursuit of Data Science.
+
+## What will be covered
+
+The Sidecar Academy consists of (1) [Data Science Prep Course](https://ldssa.github.io/wiki/DS%20Prep%20Course/Data-Science-Prep-Course/) and (2) [Lisbon Data Science Starters Academy (LDSSA)](https://ldssa.github.io/wiki/Starters%20Academy%20(LDSSA)/01-Starters-Academy-(Course)/)
+
+Proficiency in Python, and basic knowledge of Software engineering is required to adequately follow LDSSA's learning materials. As such, we will first cover the Prep Course to provide you with Python skills and other requirements. We'll then go through the LDSSA.
+
```

Comment:
I would add the following information (from https://github.com/LDSSA/sidecar-academy/wiki)

The Unofficial Sidecar Academy lasts approximately XX weeks of part-time work over the course of XX months. Everything is done remotely via GitHub, Slack, and the material's self-grading system. You do not need to quit your primary responsibility to take part in this course.

The curriculum can be referenced via the [curriculum-development repo](https://github.com/LDSSA/curriculum-development).

Reply:
Good point! Will add it